cli/cmd/cdk.go

```diff
@@ -97,8 +97,28 @@ func (c *cliState) Honeyvent(ctx context.Context, in *cdk.HoneyventRequest) (*cd
 	return &cdk.Reply{}, nil
 }
 
-// Serve will start the CDK gRPC server at the provided port and log level
-func (c *cliState) Serve(target string) error {
+// Serve will start the CDK gRPC server
+func (c *cliState) Serve() error {
+	// Start the gRPC server for components to communicate back
+	const max_attempts = 5
```

Comment:
by default, we don't use snake_case variables:
```suggestion
	const maxAttempts = 20
```
Also, maybe more attempts?